Core/Core/Logging/SpeckleLog.cs

```diff
@@ -227,6 +230,18 @@ SpeckleLogConfiguration logConfiguration
     return logger;
   }
 
+  public static void OpenCurrentLogFolder()
+  {
+    try
+    {
+      Process.Start(_logFolderPath);
+    }
+    catch
```

Comment:
Please can we ensure we log the exception 
![image](https://github.com/specklesystems/speckle-sharp/assets/45512892/524963b7-0cb8-451b-8de0-0a3f68c98e0f)